cfgov/v1/__init__.py

```diff
@@ -35,12 +39,32 @@ def environment(**options):
         'choices_for_page_type': ref.choices_for_page_type,
         'is_blog': ref.is_blog,
         'get_page_state_url': share.get_page_state_url,
+        'cfgov_rich_text': external_links,
```

Comment:
This could be very confusing; we are naming the function `cfgov_rich_text ` but it's actually converting external links.